pkg/gofr/datasource/mongo/mongo_test.go

```diff
@@ -502,8 +504,9 @@ func Test_HealthCheck(t *testing.T) {
 			Message: "duplicate key error",
 		}))
 
-		resp := cl.HealthCheck()
+		resp, err := cl.HealthCheck()
 
+		assert.Equal(t, err, errors.New(errStatusDown))
```

Comment:
Either you define a sentinel error, it could be interesting, so you could use

```go
assert.ErrorIs(t, err, ErrStatusDown)
```

See https://github.com/gofr-dev/gofr/pull/780#discussion_r1666685504

Reply:
I will open a PR to enable testifylint on updated code (not whole codebase)

https://github.com/gofr-dev/gofr/issues/787#issuecomment-2209629728